Sync/Synchronizers/TabsSynchronizer.swift

```diff
@@ -199,7 +199,7 @@ extension RemoteTab {
             "title": title,
             "icon": icon?.absoluteString as Any? ?? NSNull(),
             "urlHistory": tabHistory,
-            "lastUsed": millisecondsToDecimalSeconds(lastUsed)
+            "lastUsed": millisecondsToSeconds(lastUsed)
```

Comment:
Storage format 5 documents this as being a string representation, not a numeric value:

https://mozilla-services.readthedocs.io/en/latest/sync/objectformats.html#tabs

The storage version at the top of this file hasn't changed, so I believe this should remain a string, even if you choose to truncate the number of seconds.

Reply:
Cross-linked to the archaeology:

https://github.com/mozilla/application-services/issues/3892#issuecomment-865283813

Reply:
@rnewman 👋🏻 hey, nice to see you!

In practice, other current clients are setting this to be an integer rather than a string already, e.g. [here](https://searchfox.org/mozilla-central/rev/c114db74a92cf15096dfda02255e125949b0e070/services/sync/modules/engines/tabs.js#209) for Desktop Firefox. So I have high confidence that this change isn't going to break anything in practice, and that for better or worse, it's the docs that should be considered incorrect here.

Reply:
If you're willing to geeently bend backward compatibility, rock on — just be aware that there are clients out there who are likely to be expecting strings. Postel's Law applies, as always.

Reply:
Ack, and thanks. Any such clients are already not syncing correctly with Desktop and haven't been for some time, so I do not expect any fallout from this in practice (and will gladly wear responsibility for that call if we do see any).